app/controllers/assessment/assessments_controller.rb

```diff
@@ -3,13 +3,16 @@ class Assessment::AssessmentsController < ApplicationController
   load_and_authorize_resource :assessment, only: [:reorder, :stats, :access_denied]
   before_filter :load_general_course_data, only: [:show, :index, :new, :edit, :access_denied, :stats, :overview, :listall]
 
+  include GradingsSummaryBuilder
+
   def index
     assessment_type = params[:type]
     selected_tags = params[:tags]
 
     display_columns = {}
-    time_format =  @course.time_format(assessment_type)
+    time_format = @course.time_format(assessment_type)
     paging = @course.paging_pref(assessment_type)
+    pdf_export = @course.pdf_export(assessment_type).display
```

Comment:
instead of using .display, change the method to: pdf_export_enabled?, and call .display in the method, to improve code  readability.